src/common/options/mds.yaml.in

```diff
@@ -1105,6 +1105,16 @@ options:
   services:
   - mds
   with_legacy: true
+# Max number of slow ops to track
+- name: mds_op_history_slow_op_size
+  type: uint
+  level: advanced
+  default: 20
+# track the op if over this threshold
+- name: mds_op_history_slow_op_threshold
```

Comment:
I guess description can be
desc: duration of a MDS op to be considered as a historical slow MDS op